README.md

```diff
@@ -107,6 +107,42 @@ RSpec.describe "Posts Controller", type: :request do
       response 200, description: "success"
     end
   end
+
+  # Path Update Object
+  path '/posts/{post_id}' do
+    # Parameter Object for header
+    parameter 'Content-Type', {in: :header, type: :string}
+    let(:'Content-Type') { 'application/json' }
+	# ^same can be used for *Authorization* header
+	
+    # Parameter Object
+    parameter "post_id", {in: :path, type: :integer}
+    let(:post_id) { 1 }
+    
+    # Parameter Object for Body
+    parameter "body", {in: :body, required: true, schema: {
+	    consumes: 'application/json',
```

Comment:
I don't think `consumes` can be in the `schema`... feel free to point me to the right spot in the docs.

Also could you use two spaces for indenting please?